modules/olmv1-finding-operators-to-install.adoc

```diff
@@ -20,14 +20,14 @@ After you add a catalog to your cluster, you can query the catalog to find Opera
 +
 [source,terminal]
 ----
-$ oc -n openshift-catalogd port-forward svc/catalogd-catalogserver 8080:80
+$ oc -n openshift-catalogd port-forward svc/catalogd-catalogserver 8080:443
 ----
 
 . Download the catalog's JSON file locally by running the following command:
 +
 [source,terminal]
 ----
-$ curl -L http://localhost:8080/catalogs/<catalog_name>/all.json \
+$ curl -L https://localhost:8080/catalogs/<catalog_name>/all.json \
```

Comment:
@joelanford Would we still need to set up the port forwarding? Or could we skip that step as well?

Reply:
And would the whole command look like this?
```
. Download the catalog's JSON file locally by running the following command:
+
[source,terminal]
----
$ curl -L $(oc get clustercatalog <catalog_name> -o jsonpath='{.status.contentURL}') \
  -C - -o /<path>/<catalog_name>.json
----

Reply:
Hmm. Still need port forwarding. But that poses another problem for docs because we're _also_ not yet committing to the names of the services.

How critical is it for us to document any of this? I wonder if we could for now say to use:
```
opm render $(oc get clustercatalog -o jsonpath='{.status.resolvedSource.image.resolvedRef}')
```

Reply:
@michaelryanpeter @joelanford 
here is what I did just:
on one terminal
```console
[root@preserve-olm-env2 openshift-tests-private]# oc -n openshift-catalogd port-forward svc/catalogd-catalogserver 8080:443
Forwarding from 127.0.0.1:8080 -> 8083
Forwarding from [::1]:8080 -> 8083
Handling connection for 8080
```
on the other terminal:
```console
[root@preserve-olm-env2 bin]# curl -L https://localhost:8080/catalogs/clustercatalog-75493/all.json -C - -o ./clustercatalog-75493.json
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
curl: (60) SSL certificate problem: self signed certificate in certificate chain
More details here: https://curl.haxx.se/docs/sslcerts.html

curl failed to verify the legitimacy of the server and therefore could not
establish a secure connection to it. To learn more about this situation and
how to fix it, please visit the web page mentioned above.
[root@preserve-olm-env2 bin]# ls -l ./clustercatalog-75493.json
ls: cannot access './clustercatalog-75493.json': No such file or directory
[root@preserve-olm-env2 bin]# curl -L -k https://localhost:8080/catalogs/clustercatalog-75493/all.json -C - -o ./clustercatalog-75493.json
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100  4342  100  4342    0     0  30152      0 --:--:-- --:--:-- --:--:-- 30152
[root@preserve-olm-env2 bin]# ls -l ./clustercatalog-75493.json
-rw-r--r--. 1 root root 4342 Sep 25 11:29 ./clustercatalog-75493.json
```
so, we need `-k` option.

I also tried `$(oc get clustercatalog <catalog_name> -o jsonpath='{.status.contentURL}')`.
```console
[root@preserve-olm-env2 bin]# curl -L -k $(oc get clustercatalog clustercatalog-75493 -o jsonpath='{.status.contentURL}') -C - -o ./clustercatalog-75493.json
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0curl: (6) Could not resolve host: catalogd-catalogserver.openshift-catalogd.svc
[root@preserve-olm-env2 bin]# 
it does not work. after porting-forward, we can not still use the status.contentURL, and have to use the `localhost:8080`
```

Reply:
FYI https://operator-framework.github.io/operator-controller/Tasks/exploring-available-packages/

Reply:
@joelanford I created a spike to investigate the issue you raised here for OCP 4.18. https://issues.redhat.com/browse/OSDOCS-12130

Reply:
I am equally concerned as @joelanford about documenting the URL . I would like to fix this in both upstream and here for 4.18 as we do not want to give the impression that the URL is part of our API